com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl

```diff
@@ -156,6 +156,78 @@ TEXTURE2D(_PrevExposureTexture);
 
 #include "Packages/com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariablesXR.cs.hlsl"
 
+// In HDRP, all material samplers have the possibility of having a mip bias.
```

Comment:
Let's add a define in ShaderConfig.hlsl to control if we enable those Macro or not. 
Also good to check with our performance test if we have any difference for our console test

Reply:
Done, check the ShaderConfig.cs and let me know if that works. I tested it locally and works well too.